contracts/libraries/Math.sol

```diff
@@ -1,16 +1,19 @@
 pragma solidity 0.8.13;
```

Comment:
[Question] We can remove this contract as we are now using openzeppelin ones.

Reply:
does forge still use it?

Reply:
no test cases are using this.